File: 2st/Yeonwoo/홀수 vs 짝수.py
def solution(num_list):
    answer = 0
    even = 0
    odd = 0
    for i, num in enumerate(num_list):
        if i%2==0:
            even += num
        elif i%2==1:
            odd += num
    answer = max(even, odd)
    return answer

File: 3st/Juhee/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int odd = 0;
        int even = 0;

        for (int i = 0; i < num_list.length; i++) {
            if (i % 2 == 0) {
                even += num_list[i];
            } 
            else {
                odd += num_list[i];
            }
        }

        return (odd > even) ? odd : even;
    }
}